packages/components/src/InputTime/InputTime.test.tsx

```diff
@@ -1,185 +1,37 @@
 import React from "react";
-import { fireEvent, render } from "@testing-library/react";
+import { fireEvent, render, screen } from "@testing-library/react";
 import { CivilTime } from "@std-proposal/temporal";
 import { InputTime } from ".";
 
-it("renders a InputTime", () => {
```

Comment:
I like seeing all the deleted inline snapshots. I think they were the items giving me issues in the other stream of work, so this might make my life easier! :)